### DISCUSSION TITLE
Match multiple columns from same input string

### DISCUSSION THREAD

**Discussion by nolawnchairs** [prisma/prisma#18302] [Q&A]
Hi,

I'm trying to query if the title field contains a string or if the hash field equals the string, but I get zero results from this query:

```ts
this.db.video.count({
  where: {
    OR: [
      { title: { contains: search } },
      { hash: search },
    ],
  },
})
```

What is wrong with my query?

**Comment by nurul3101** [comment]
Hello @nolawnchairs 👋 

Can you please share your schema file or the video model? 
Also, which database and prisma version are you using?

**Reply by nurul3101** [reply]
@nolawnchairs Do let us know in case you are facing any issues

**Reply by nurul3101** [reply]
@nolawnchairs 
It would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏
